docs/msbuild/common-msbuild-project-items.md

```diff
@@ -125,6 +125,14 @@ In [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.m
 | Visible | Optional boolean. Indicates whether to display the file in **Solution Explorer** in [!INCLUDE[vsprvs](../code-quality/includes/vsprvs_md.md)]. |
 | CopyToOutputDirectory | Optional string. Determines whether to copy the file to the output directory. Values are:<br /><br /> 1.  Never<br />2.  Always<br />3.  PreserveNewest |
 
+### AssemblyMetadata
+ Represents assembly attributes to be generated as `[AssemblyMetadata(key, value)]`.
+
+| Item metadata name | Description |
+|-----------------------| - |
+| Include | Becomes the first parameter (the key) in the `AssemblyMetadataAttribute` attribute constructor. |
+| Value | Required string. Becomes the second parameter (the value) in the `AssemblyMetadataAttribute` attribute constructor. |
+
```

Comment:
```suggestion

> [!NOTE]
> This applies to projects using the .NET Core SDK only.
```

Reply:
I used the note stating that it applies to the  .NET Core SDK.
Is it possible to add a required version? (I assume a version of the SDK).

Reply:
This would apply only in SDK 5.0, right @livarcocc? It documents https://github.com/dotnet/sdk/pull/3440 which is targeted to sdk/master.

Reply:
@livarocc for input